src/debug.c

```diff
@@ -3855,31 +3165,31 @@ static void DebugAction_TimeMenu_ChangeWeekdays(u8 taskId)
     DebugAction_DestroyExtraWindow(taskId);
     switch(input)
     {
-        case DEBUG_TIME_MENU_ITEM_SUNDAY:
+        case WEEKDAY_SUN:
             daysToAdd = ((WEEKDAY_SUN - rtc->dayOfWeek) + WEEKDAY_COUNT) % WEEKDAY_COUNT;
```

Comment:
can this switch case not be simplified now?
```suggestion
            daysToAdd = ((input - rtc->dayOfWeek) + WEEKDAY_COUNT) % WEEKDAY_COUNT;
```

Reply:
> can this switch case not be simplified now?

Good catch!